helm/cas-postgres/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: cas-postgres
+description: Postgres database for the CAS team
+type: application
+version: 0.2.0
+appVersion: 0.1.0
+dependencies:
+- name: patroni
+  version: "0.18.0"
+  repository: "file://../patroni"
```

Comment:
The plan is to put this in a chart repo (even if just github), right? Do we have a card to track this?

Reply:
check out https://github.com/bcgov/cas-postgres/releases
the patroni chart is getting packaged when you do `helm package`